src/transformers/models/speech_to_text_2/configuration_speech_to_text_2.py

```diff
@@ -31,7 +31,7 @@ class Speech2Text2Config(PretrainedConfig):
     This is the configuration class to store the configuration of a [`Speech2Text2ForCausalLM`]. It is used to
     instantiate an Speech2Text2 model according to the specified arguments, defining the model architecture.
     Instantiating a configuration with the defaults will yield a similar configuration to that of the Speech2Text2
-    [facebook/s2t-small-librispeech-asr](https://huggingface.co/facebook/s2t-small-librispeech-asr) architecture.
+    [facebook/s2t-wav2vec2-large-en-de](https://huggingface.co/facebook/s2t-wav2vec2-large-en-de) architecture.
```

Comment:
`facebook/s2t-small-librispeech-asr` has `model_type": "speech_to_text`, but here is for the model `Speech2Text2` (i.e. `speech_to_text_2`)

Reply:
Great catch!